I get some errors when I'm running the tests inside a container
```
======================================================================
ERROR: test_003_pass_credentials_through_app_conf (tests.TestFlaskBoto3Clients)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/sources/tests/__init__.py", line 133, in test_003_pass_credentials_through_app_conf
    b.connections
  File "/sources/flask_boto3.py", line 95, in connections
    ctx.boto3_cns = self.connect()
  File "/sources/flask_boto3.py", line 38, in connect
    sess = boto3.session.Session(**sess_params)
  File "/usr/local/lib/python3.6/site-packages/boto3/session.py", line 80, in __init__
    self._setup_loader()
  File "/usr/local/lib/python3.6/site-packages/boto3/session.py", line 112, in _setup_loader
    self._loader = self._session.get_component('data_loader')
  File "/usr/local/lib/python3.6/site-packages/botocore/session.py", line 695, in get_component
    return self._components.get_component(name)
  File "/usr/local/lib/python3.6/site-packages/botocore/session.py", line 889, in get_component
    self._components[name] = factory()
  File "/usr/local/lib/python3.6/site-packages/botocore/session.py", line 179, in <lambda>
    lambda:  create_loader(self.get_config_variable('data_path')))
  File "/usr/local/lib/python3.6/site-packages/botocore/session.py", line 259, in get_config_variable
    elif self._found_in_config_file(methods, var_config):
  File "/usr/local/lib/python3.6/site-packages/botocore/session.py", line 280, in _found_in_config_file
    return var_config[0] in self.get_scoped_config()
  File "/usr/local/lib/python3.6/site-packages/botocore/session.py", line 352, in get_scoped_config
    raise ProfileNotFound(profile=profile_name)
botocore.exceptions.ProfileNotFound: The config profile (default) could not be found
```

it seems that we'll need to create a temp creds file and pass its path to boto3.

--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -1,3 +1,5 @@
+import os
+
 from unittest import TestCase
 from mock import patch
 from nose.tools import assert_is_instance, assert_list_equal, eq_
@@ -7,12 +9,23 @@ from flask import _app_ctx_stack as stack
 from flask.ext.boto3 import Boto3
 
 
+def create_aws_mock_config():
+    aws_dir = os.path.expanduser('~/.aws')
+    aws_config = aws_dir + '/config'
+    if not os.path.exists(aws_dir):
+        os.makedirs(aws_dir)
+    if not os.path.isfile(aws_config):
+        with open(aws_config, 'w') as f:
+            f.write('[default]')
+
+
 @patch('boto3.session.Session.resource')
 class TestFlaskBoto3Resources(TestCase):
 
     def setUp(self):
         self.app = Flask('unit_tests')
         self.app.config['BOTO3_REGION'] = 'eu-west-1'
+        create_aws_mock_config()
 
     def test_001_populate_application_context(self, mock_resource):
         self.app.config['BOTO3_SERVICES'] = ['s3', 'sqs']
@@ -102,6 +115,7 @@ class TestFlaskBoto3Clients(TestCase):
     def setUp(self):
         self.app = Flask('unit_tests')
         self.app.config['BOTO3_REGION'] = 'eu-west-1'
+        create_aws_mock_config()
 
     def test_001_populate_application_context(self, mock_client):
         self.app.config['BOTO3_SERVICES'] = ['codebuild', 'codedeploy']

